flink-container/src/test/java/org/apache/flink/container/entrypoint/ClassPathJobGraphRetrieverTest.java

```diff
@@ -53,10 +68,69 @@
 	@Rule
 	public final TemporaryFolder temporaryFolder = new TemporaryFolder();
 
+	@ClassRule
+	public static final TemporaryFolder JOB_DIRS = new TemporaryFolder();
+
 	private static final String[] PROGRAM_ARGUMENTS = {"--arg", "suffix"};
 
+	/*
+	 * The directory structure used to test
+	 *
+	 * userDirHasEntryClass/
+	 * 					  |_jarWithEntryClass
+	 * 					  |_jarWithoutEntryClass
+	 * 					  |_textFile
+	 *
+	 * userDirHasNotEntryClass/
+	 * 						 |_jarWithoutEntryClass
```

Comment:
nit: the texts are not aligned.

Reply:
I think it is the github's problem.

Reply:
I guess not. It's also not aligned in IDE. 
Looks to me it is using a mixture of tabs and spaces, and the last but one char is a tab in the upper while it is a space in the lower.